cms/scans.py

```diff
@@ -32,7 +32,7 @@ def parser_selscan_file_conversion(parser=argparse.ArgumentParser()):
             Process a bgzipped-VCF (such as those included in the Phase 3 1000 Genomes release) into a gzip-compressed
             tped file of the sort expected by selscan. 
     """
-    parser.help = """This function converts an input VCF into a TPED file usable by selscan.
```

Comment:
I have the above change in pull request #26. You'll see it if you sync with the master branch.